src/99-appendices/01-contributors.md

```diff
@@ -30,6 +30,7 @@ The following individuals have contributed to the Brain Imaging Data Structure
 ecosystem (in alphabetical order). If you contributed to the BIDS ecosystem and
 your name is not listed, please add it.
 
+-   Fidel Alfaro Almagro 💬📖💡🔌
 -   Stefan Appelhoff 📖💬🤔🐛💡💻👀⚠️📢
```

Comment:
```suggestion
-   Stefan Appelhoff 📖💬🤔🐛💡💻👀⚠️📢✅🔧🔌📝
```

Reply:
all set, thank you for updating the credit!